docker-compose.verifier.yml

```diff
@@ -0,0 +1,102 @@
+      # Geth Submission Queuer (needs Postgres vars above)
+      - IS_SEQUENCER_STACK=                                                                         # (Required) Set if this is queueing Geth submissions for a sequencer (and not _just_ a verifier)
+      - GETH_SUBMISSION_QUEUER_PERIOD_MILLIS                                                        # The period in millis at which the GethSubmissionQueuer should attempt to queue an L2 Geth submission (defaults to 10,000)
+      # Queued Geth Submitter (needs Postgres & L2 Node vars above)
+      - SUBMIT_TO_L2_PRIVATE_KEY=0x8dcd66caf66a67f2ef66d808971efc54560eac459092aee43c07d76efae8a032 # (Required) The PK to use to sign batches to send to geth (deprecated)
+      - QUEUED_GETH_SUBMITTER_PERIOD_MILLIS                                                         # The period in millis at which the QueuedGethSubmitter should attempt to send L2 Geth submissions (defaults to 10,000)
+      # Fraud Detector
+      - FRAUD_DETECTOR_PERIOD_MILLIS                                                                # The period in millis at which the FraudDetector should run (defaults to 10,000)
+      - REALERT_ON_UNRESOLVED_FRAUD_EVERY_N_FRAUD_DETECTOR_RUNS                                     # The number of runs after which a detected fraud, if still present, should re-alert (via error logs) (defaults to 10)
+      # Which Services to run (respective vars must be configured above)
+      - RUN_L1_CHAIN_DATA_PERSISTER=1                                                               # Set to anything to run L1 Chain Data Persister
+      - RUN_L2_CHAIN_DATA_PERSISTER=1                                                               # Set to anything to run L2 Chain Data Persister
+      - RUN_GETH_SUBMISSION_QUEUER=1                                                                # Set to anything to run Geth Submission Queuer
+      - RUN_QUEUED_GETH_SUBMITTER=1                                                                 # Set to anything to run Queued Geth Submitter
+      - RUN_FRAUD_DETECTOR=1                                                                        # Set to anything to run Fraud Detector
+      # Sequencer-only Services
+      - RUN_CANONICAL_CHAIN_BATCH_CREATOR=                                                          # Set to anything to run Canonical Chain Batch Creator
+      - RUN_CANONICAL_CHAIN_BATCH_SUBMITTER=                                                        # Set to anything to run Canonical Chain Batch Submitter & Finalizer
+      - RUN_STATE_COMMITMENT_CHAIN_BATCH_CREATOR=                                                   # Set to anything to run State Commitment Chain Batch Creator
+      - RUN_STATE_COMMITMENT_CHAIN_BATCH_SUBMITTER=                                                 # Set to anything to run State Commitment Chain Batch Submitter & Finalizer
```

Comment:
When you leave these unset it doesn't cause any errors or anything right?

Reply:
Nope -- just left them in for more info.

Reply:
Cool!!!